app/views/appointments/customer.html.erb

```diff
@@ -1,51 +1,27 @@
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">Dear <%= @appointment.name %>,</p>
-
-<% if @appointment.updated? %>
-  <p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin:
-  15px 0;font-size: 16px;line-height: 1.315789474;">Your appointment details were updated.</p>
-<% end %>
+<h1 style="color: #0B0C0C !important;font-family: Helvetica, Arial, sans-serif;margin: 15px 0;font-weight: 700;font-size: 20px;line-height: 1.111111111;padding: 15px 0;">
+  <% if @appointment.updated? %>
+    We’ve updated your appointment
+  <% else %>
+    We’ve booked your appointment
+  <% end %>
+</h1>
 
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">Thank you for booking an
-appointment with Pension Wise. We can confirm your appointment will be at <%=
-@appointment.proceeded_at.to_s(:govuk_date) %>. The appointment will take place
-at <%= @appointment.location_name %> and will be with <%= @appointment.guider_name.split.first %>.</p>
+<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px 0;font-size: 16px;line-height: 1.315789474;">
+  Dear <%= @appointment.name %>,
+</p>
 
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">
-  <% @appointment.address_lines.each do |line| %>
-    <%= line %><br>
+<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px 0;font-size: 16px;line-height: 1.315789474;">
+  <% if @appointment.updated? %>
+    Your appointment details have been updated.
+  <% else %>
+    Thank you for booking a Pension Wise appointment.
   <% end %>
 </p>
 
-<h3 style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif">Preparing for your appointment</h3>
-
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">To make your appointment as useful
-as possible, it would be helpful to know the following information:</p>
+<%= render 'shared/email/html/appointment_details' %>
```

Comment:
I thinking splitting the HTML / text parts into separate folders is not necessary. If you keep them in the same folder with the same filenames and suffixed with the correct type (text.erb or html.erb) they should be correctly inferred.